lib/src/services/analytics_service.dart

```diff
@@ -34,7 +34,6 @@ class AnalyticsService {
     await _analytics.waitForLastPing(
       timeout: const Duration(milliseconds: 200),
     );
-    _analytics.close();
```

Comment:
@ferrarafer this line is what caused the http issue we've been seeing. It closes the analytics and in the comments of this function it says, after this call we should not call analytics again.